pyproject.toml

```diff
@@ -16,14 +16,20 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "^3.7"
-requests = "^2.26.0"
+httpx = "^0.20.0"
```

Comment:
@J0

Reply:
> It would be interesting to unify all supabase python libs into using httpx.

`supabase-py` still contains a mix of `requests` and `httpx` the other libs only use `httpx`

Do you mean unifying the use of the same version?

Reply:
Well, I meant the use of httpx. The version depends on the other dependencies. Would it possible to replace the use of requests in `supabase-py` by `httpx`? @J0

Reply:
Technically it is simple and the changes are few because the `httpx` api pretends to be similar to the `requests` api

Reply:
If you have time while this is being approved, you resolve this and the typing issues.

Reply:
Hey all, sorry for the delays on this. Will revert by end of day.

Reply:
Sure, I am for moving to httpx. Let's go with that -- will update the roadmap accordingly.